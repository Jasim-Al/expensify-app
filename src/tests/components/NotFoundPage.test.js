import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should show ExpensesDashboardPage correctley', () => {
    let wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();
});
