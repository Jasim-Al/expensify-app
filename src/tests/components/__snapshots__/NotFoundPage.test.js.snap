// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show ExpensesDashboardPage correctley 1`] = `
<div>
  <p>
    404! 
    <Link
      to="/"
    >
      Go Home
    </Link>
  </p>
</div>
`;
